### DISCUSSION TITLE
network/summary

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#255] [General]
# network/summary

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/network/summary.html

**Comment by zjyyzhengli** [comment]
图3-0中 “3.4 内核参数优化实践” 这个提纲随着对应内容的优化，也需要同步更新一下了。

**Reply by isno** [reply]
嗯，这一块到一月份再改。这几天得集中精力把最后一章写完
